strawberry/subscriptions/protocols/graphql_transport_ws/handlers.py

```diff
@@ -263,6 +263,10 @@ async def handle_async_results(
             return
 
     async def handle_complete(self, message: CompleteMessage) -> None:
+        timeout_task = self.connection_init_timeout_task
+        if timeout_task and not timeout_task.done() and not timeout_task.cancelled():
+            timeout_task.cancel()
```

Comment:
@DoctorJohn could you take a look at this?

Reply:
If I understand the protocol correctly, a `Complete` message can only be send the the server after the connection has been acknowledged by the server, in which case the timeout task takes care of itself.

I think the intention here was to explicitly cancel the timeout task when it's not needed anymore. We could still do that, but I don't feel like this method is the right place to do so. Maybe `handle_connection_init` would be better. If we want to cancel it, we would probably have to await the cancelled task (see my other comment).

Reply:
I actually did did because I saw some errors when running tests. I just removed both this and the code inside the channels handler and I'm not getting the error anymore.

Maybe it was something that I actually caused because of something wrong while taking over the existing PR? Not sure! =P. But either way, I just removed the code and if it is actually required, we can always add it again in the future